wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -886,13 +900,33 @@ instance ToSchema Account where
             & ("walletId"  --^ "Id of the wallet this account belongs to.")
           )
 
+instance ToSchema AccountAddresses where
+    declareNamedSchema =
+        genericSchemaDroppingPrefix "aca" (\(--^) props -> props
+            & ("addresses" --^ "Public addresses pointing to this account.")
+          )
+
+instance ToSchema AccountBalance where
+    declareNamedSchema =
+        genericSchemaDroppingPrefix "acb" (\(--^) props -> props
+            & ("amount"    --^ "Available funds, in Lovelace.")
+          )
+
 instance Arbitrary Account where
     arbitrary = Account <$> arbitrary
                         <*> arbitrary
                         <*> arbitrary
                         <*> pure "My account"
                         <*> arbitrary
 
+instance Arbitrary AccountAddresses where
+    arbitrary =
+        AccountAddresses <$> arbitrary
```

Comment:
with deriving strategies, this is `deriving newtype (Arbitrary)` :smile:

Reply:
I am always (without any good reasons) afraid of deriving newtypes automagically :p ...